resources/public/api/conf/2.0/schemas/retrieveBalanceAndTransactions/financialDetails.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema":"https://json-schema.org/draft/2019-09/schema",
+  "type":"object",
+  "properties":{
+    "financialDetails":{
+      "type":"array",
+      "items":{
+        "type":"object",
+        "description":"A list of the items that make up the overall transaction. The items may include subitems.",
+        "properties":{
+          "taxYear":{
+            "description":"The tax year to which the item relates, in the format YYYY-YY",
+            "type":"string",
+            "example":"2020-21"
+          },
+          "chargeDetail":{
+            "type":"object",
+            "properties":{
+              "documentId":{
+                "type":"string",
+                "description":"A unique ID is used to reference the document of which this is a subitem.\nMust conform to the regular expression: ^[0-9a-zA-Z]{1,30}$",
+                "examples":"XAIS12345678910"
```

Comment:
General note for this file: I think 'examples' might need to be made singular, like you've done it on line 14